### DISCUSSION TITLE
docs/ninjutsu/up-ranking-ninja/self-improvement

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#11] [General]
# docs/ninjutsu/up-ranking-ninja/self-improvement

即便是摸鱼也要卷赢别人。S属性大爆发，Study!!!

https://996.ninja/docs/ninjutsu/up-ranking-ninja/self-improvement

**Comment by BBFTZ** [comment]
这个有点意思

**Comment by AjieEL** [comment]
我是一颗卷心菜

**Reply by MingGH** [reply]
卷心菜你好

**Comment by pointm** [comment]
绝佳的互联网兔子洞，上班时候可有的摸了~呱~

**Reply by MingGH** [reply]
哈哈哈哈哈
